<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>224</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UDP Packet Forwarder</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="PlayPushButton">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PausePushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClosePushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="DelayLabel">
      <property name="text">
       <string>Delay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="DelaySpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MillSecondLabel">
      <property name="text">
       <string>(ms)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>68</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="NumPcktInLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string># pckt in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="NumInPcktLCDNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="NumPcktInLabel_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string># pckt Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="NumOutPcktLCDNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="NumPcktInLabel_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string># pckt Qed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="NumPcktInQLCDNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="SrcLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>48</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DstLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>68</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>20</y>
     <width>98</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="IPLabel">
      <property name="text">
       <string>IP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SrcIPLineEdit">
      <property name="text">
       <string>225.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DstIPLineEdit">
      <property name="text">
       <string>225.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>56</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="PortLabel">
      <property name="text">
       <string> Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SrcPortLineEdit">
      <property name="text">
       <string>50120</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DstPortLineEdit">
      <property name="text">
       <string>60120</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SrcIPLineEdit</tabstop>
  <tabstop>SrcPortLineEdit</tabstop>
  <tabstop>DstIPLineEdit</tabstop>
  <tabstop>DstPortLineEdit</tabstop>
  <tabstop>DelaySpinBox</tabstop>
  <tabstop>PlayPushButton</tabstop>
  <tabstop>PausePushButton</tabstop>
  <tabstop>StopPushButton</tabstop>
  <tabstop>ClosePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
